$NetBSD: patch-aa,v 1.5 2021/11/28 05:22:15 pho Exp $

* Change the hard-coded plugin path.
* NetBSD requires -lossaudio to use OSS.
* Honor LDFLAGS to support RELRO.

--- Makefile.orig	2004-08-19 20:07:27.000000000 +0000
+++ Makefile
@@ -8,14 +8,14 @@ export PACKAGE LOCALEDIR
 
 GTK_CONFIG = pkg-config gtk+-2.0
 
-PLUGIN_DIR ?= /usr/local/lib/gkrellm2/plugins
+PLUGIN_DIR ?= $(PREFIX)/lib/gkrellm2/plugins
 GKRELLM_INCLUDE = -I/usr/local/include
 
 GTK_CFLAGS = `$(GTK_CONFIG) --cflags` 
 GTK_LIB = `$(GTK_CONFIG) --libs`
 
 FLAGS = -O2 -Wall -fPIC $(GTK_CFLAGS) $(GKRELLM_INCLUDE)
-LIBS = $(GTK_LIB)
+LIBS = $(GTK_LIB) $(LIBOSSAUDIO)
 LFLAGS = -shared
 
 OBJS = volume.o mixer.o oss_mixer.o
@@ -40,7 +40,7 @@ all:	volume.so
 	(cd po && ${MAKE} all )
 
 volume.so: $(OBJS)
-	$(CC) $(OBJS) -o volume.so $(LIBS) $(LFLAGS)
+	$(CC) $(OBJS) -o volume.so $(LIBS) $(LFLAGS) $(LDFLAGS)
 
 clean:
 	rm -f *.o core *.so* *.bak *~
